adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDDFunctions.scala

```diff
@@ -19,7 +19,7 @@ package org.bdgenomics.adam.rdd.read
 
 import java.io.StringWriter
 
-import htsjdk.samtools.{ SAMFileHeader, SAMTextHeaderCodec, SAMTextWriter, ValidationStringency }
+import htsjdk.samtools.{ TextCigarCodec, SAMFileHeader, SAMTextHeaderCodec, ValidationStringency, SAMTextWriter }
```

Comment:
haha funny you should ask; I [just complained to JetBrains](https://youtrack.jetbrains.com/issue/IDEA-144185) about IntelliJ reverting my disabling of the wildcard-imports when it exceeds 4, the preferences-search making it hard to find that setting, etc.

I'm fine to wildcard it if that's what you prefer.